test/tests/api-cit/v2_0/workflows_tests.py

```diff
@@ -154,35 +166,42 @@ def test_workflows_graphs_put(self):
     def test_workflows_library_id_get(self):
         # """ Testing GET:/workflows/graphs/injectableName"""
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        self.assertEqual(200, self.__client.last_response.status)
-        rawj = loads(self.__client.last_response.data)
+        self.assertEqual(200, self.__get_result().status)
+        rawj = self.__get_data()
         self.assertEqual(self.workflowDict.get('friendlyName'), str(rawj[0].get('friendlyName')))
 
     @depends(after='test_workflows_library_id_get')
     def test_workflows_graphs_name_put(self):
         # """Testing PUT:/workflows/graphs"""
         # Test updating a graph
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        rawj = loads(self.__client.last_response.data)
+        rawj = self.__get_data()
         self.assertEqual(self.workflowDict.get('friendlyName'), str(rawj[0].get('friendlyName')))
         Api().workflows_put_graphs(body=self.workflowDict2)
-        self.assertEqual(201, self.__client.last_response.status)
+        self.assertEqual(201, self.__get_result().status)
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        rawj = loads(self.__client.last_response.data)
+        rawj = self.__get_data()
         self.assertEqual(self.workflowDict2.get('friendlyName'), str(rawj[0].get('friendlyName')))
 
     @depends(after='test_workflows_graphs_name_put')
     def test_workflows_graphs_delete(self):
         # """Testing DELETE:/workflows/graphs/injectableName"""
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        rawj = loads(self.__client.last_response.data)
+        rawj = self.__get_data()
         self.assertEqual(self.workflowDict2.get('friendlyName'), str(rawj[0].get('friendlyName')))
         Api().workflows_delete_graphs_by_name(self.workflowDict.get('injectableName'))
-        self.assertEqual(204, self.__client.last_response.status)
+        self.assertEqual(204, self.__get_result().status)
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        self.assertEqual(0, len(loads(self.__client.last_response.data)))
+        self.assertEqual(0, len(self.__get_data()))
+
+    @depends(after=['test_workflows_graphs_name_put', 'test_delete_all_active_workflows'])
+    def test_node_workflows_post(self):
+        logs.info('************************* test_node_workflows_post ')
```

Comment:
nit: I recall throwing these in with all the '*'s in order to find them in the logs easier. I don't think they should stay way?